test/testfiles/bridge-udp-stale-conntrack.json

```diff
@@ -0,0 +1,32 @@
+{
+   "container_id": "udp_traffic_container",
+   "container_name": "udp_traffic_test",
+   "port_mappings": [
+       {
+           "host_ip": "127.0.0.1",
+           "container_port": 8080,
+           "host_port": 10080,
+           "range": 1,
```

Comment:
it would likely be nice to have another test with range > 1 to test the iteration is working properly, though not blocking over it.